Client/Frontend/Widgets/SnackBar.swift

```diff
@@ -136,18 +138,36 @@ class SnackBar: UIView {
         backgroundView.snp.makeConstraints { make in
             make.bottom.left.right.equalTo(self)
             // Offset it by the width of the top border line so we can see the line from the super view
-            make.top.equalTo(self).offset(1)
+            make.top.equalTo(self)//.offset(1)
         }
 
         titleView.snp.makeConstraints { make in
             make.top.equalTo(self).offset(UIConstants.DefaultPadding)
+            make.height.equalTo(UIConstants.SnackbarButtonHeight - 2*UIConstants.DefaultPadding)
```

Comment:
spacing between the symbols. `UIConstants.SnackbarButtonHeight - 2 * UIConstants.DefaultPadding`